pb-remote.py

```diff
@@ -8,9 +8,16 @@
 
 xbox_port = 9269 # "XBOX" on a phone ;)
 
-vp20compiler="/home/fox/Data/Projects/nxdk/tools/vp20compiler/vp20compiler"
+try:  
+   NXDK_DIR = os.environ["NXDK_DIR"]
+except KeyError: 
+   print("Please set the environment variable NXDK_DIR")
```

Comment:
nit: consider one of these http://stackoverflow.com/questions/5574702/how-to-print-to-stderr-in-python (as nv2a-re will probably be ran automated where stdout will be piped to logs, but stderr will be in the terminal)